pkg/otlp/config_parser.go

```diff
@@ -0,0 +1,74 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+package otlp
+
+import (
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/config/configparser"
+	"go.opentelemetry.io/collector/service/parserprovider"
+)
+
+// baseConfig is the base OTLP pipeline configuration.
+// This pipeline is extended through the datadog.yaml configuration values.
+// It is written in YAML because it is easier to read and write than a map.
+const baseConfig string = `
```

Comment:
Would it be useful if we were able to change this through the configuration ?

Reply:
We may add it in the future (at least for editing the `otlp` receiver section) but I want to do that in a separate PR